components/blitz/src/omero/gateway/model/TableDataColumn.java

```diff
@@ -0,0 +1,213 @@
+/*
+ * Copyright (C) 2016 University of Dundee & Open Microscopy Environment.
+ * All rights reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+package omero.gateway.model;
+
+/**
+ * Defines a column for a {@link TableData} object
+ * 
+ * @author Dominik Lindner &nbsp;&nbsp;&nbsp;&nbsp; <a
+ *         href="mailto:d.lindner@dundee.ac.uk">d.lindner@dundee.ac.uk</a>
+ */
+public class TableDataColumn {
+
+    /** The header */
+    private String name = "";
+
+    /** A description */
+    private String description = "";
+
+    /**
+     * The index of the column (in the original table; this doesn't have to
+     * match the index in the {@link TableData#columns} or
+     * {@link TableData#data} array, depending on which columns are loaded)
+     */
+    private int index = -1;
+
+    /** The type of data in this column */
+    private Class<?> type = null;
+
+    /**
+     * Creates a new instance
+     * 
+     * @param name
+     *            The header
+     * @param index
+     *            The index
+     * @param type
+     *            The type of data in this column
+     */
+    public TableDataColumn(String name, int index, Class<?> type) {
```

Comment:
could be replaced by this(name, index, "", class) so there is only one place to assign value